fala/apps/adviser/tests/test_search_view_function.py

```diff
@@ -70,6 +74,13 @@ def test_category_search_heading(self):
             html=True,
         )
 
+    def test_search_parameters_box_contains_only_postcode_and_categories(self):
```

Comment:
Great but a test for any categories would also be great to see to ensure that is correctly printing them into the DOM.